src/plugins/platforms/freerdp/qfreerdppeer.cpp

```diff
@@ -1411,8 +1504,8 @@ bool QFreeRdpPeer::setPointer(const POINTER_LARGE_UPDATE *largePointer, Qt::Curs
 		lpointer->cacheIndex = cacheIndex;
 		lpointer->width = largePointer->width;
 		lpointer->height = largePointer->height;
```

Comment:
These changes will break build with FreeRDP3, perhaps we should have an ifdef block with the FreeRDP version (see https://github.com/FreeRDP/FreeRDP/wiki/FreeRDP3-migration-notes)